modules/ossm-extensions-migrating-to-wasmplugin.adoc

```diff
@@ -0,0 +1,30 @@
+////
+This module included in the following assemblies:
+*service_mesh_/v2x/ossm-extensions.adoc
+////
+:_content-type: PROCEDURE
+[id="ossm-extensions-migrating-to-wasmplugin_{context}"]
+= Migrating to `WasmPlugin` resources
+
+To upgrade your WebAssembly extensions from the `ServiceMeshExtension` API to the `WasmPlugin` API, you rename your plug-in file.
+
+.Prerequisites
+
+* `ServiceMeshControlPlane` is upgraded to version 2.2 or later.
+
+[CAUTION]
+====
+Because both plugins will be called for every request, you might want to remove your existing `ServiceMeshExtension` resource before creating the new `WasmPlugin` resource. You might get undesired results having two plugins active at the same time.
+====
+
+.Procedure
+
+. Update your container image. If the plug-in is already in `/plugin.wasm` inside the container, skip to the next step.  If not:
+
+.. Ensure the plug-in file is named `plugin.wasm`. Extension files must be named `plugin.wasm`.
+
+.. Ensure the plug-in file is located in the root (/) directory. Extension files must be located in the root of the container filesystem.
```

Comment:
Just noticed one more thing. Can revise this sentence to make it active, for example, "You must store extension files in the root of the container filesystem.".